src/main/java/uk/gov/companieshouse/api/accounts/service/impl/FilingServiceImpl.java

```diff
@@ -0,0 +1,25 @@
+package uk.gov.companieshouse.api.accounts.service.impl;
+
+
+import org.springframework.stereotype.Service;
+import uk.gov.companieshouse.api.accounts.CompanyAccountsApplication;
+import uk.gov.companieshouse.api.accounts.model.filing.Filing;
+import uk.gov.companieshouse.api.accounts.service.FilingService;
+import uk.gov.companieshouse.logging.Logger;
+import uk.gov.companieshouse.logging.LoggerFactory;
+
+@Service
+public class FilingServiceImpl implements FilingService {
+
+    private static final Logger LOGGER = LoggerFactory
+        .getLogger(CompanyAccountsApplication.APPLICATION_NAME_SPACE);
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public Filing generateAccountFiling(String transaction, String accountEntity) {
```

Comment:
We should remove these method variables as they aren't being used yet. Same applies for the interface